src/graph.c

```diff
@@ -134,6 +128,7 @@ void add_edge(Graph *graph, uint32_t beg, uint32_t end, uint32_t edges_count)
   graph->adjacency_matrix[beg][end] += edges_count;
   graph->out_degrees[beg]++;
   graph->in_degrees[end]++;
```

Comment:
```suggestion
  graph->out_degrees[beg] += edges_count;
  graph->in_degrees[end] += edges_count;
```

Teraz zuważyłam, że tutaj w sumie powinno być `+= edges_count` zamiast `++`. Wcześniej każda krawędź w sumie była dodawana osobno przy wczytywaniu z konsoli czy plików, więc to nie miało znaczenia, ale powinno być w ten sposób.